tests/bluetooth/bsim/audio/src/common.c

```diff
@@ -9,6 +9,7 @@
 
 extern enum bst_result_t bst_result;
 struct bt_conn *default_conn;
+atomic_t flag_connected;
```

Comment:
Out of scope for this PR I think, but the connected and disconnected callbacks would be more flexible, i.e. allow for multiple connections, if we have
```
struct bt_conn *conns[CONFIG_BT_MAX_CONN
```

Reply:
While the current macros treat `atomic_t flag_connected;` as a boolean, it is in fact just an integer, and we could do `atomic_inc` to use this as the number of connected devices if we wanted instead.